DataFormats/L1TrackTrigger/BuildFile.xml

```diff
@@ -7,6 +7,7 @@
 <use name="DataFormats/TrackerCommon"/>
 <use name="FWCore/MessageLogger"/>
 <use name="FWCore/Utilities"/>
+<use name="hls"/>
```

Comment:
the need for this is introduced in #34550 - so should be included there rather than in a separate PR

Reply:
Yes, that's what will ultimately happen (we hope). We are seeing some strange changes in the DQM monitoring tests in #34550, so we are using this draft PR to try to track them down. Because there were no changes due to the non-L1TrackTrigger build files we decided to check the inclusion of the HLS library.